cubic-server/Player.hpp

```diff
@@ -119,6 +145,7 @@ class Player : public Entity
     uint8_t _gamemode;
     TickClock _keepAliveClock;
     bool _isFlying;
+    std::unordered_map<Position2D, ChunkStateHolder> _chunks;
```

Comment:
each player has a map of rendered or sended chunks
each dimension has a map of loading chunk